packages/custom/icu/public/components/sidepane/sidepane.directive.js

```diff
@@ -552,12 +545,14 @@ directive('icuSidepane', function() {
     };
 
     $scope.clearResults = function(){
-        SearchService.clearResults();
-        $scope.issues = $scope.issues.map(function(issue){
-            issue.length = 0;
-            return issue;
-        });
-        return $scope.issues;
+      SearchService.refreshQuery('');
```

Comment:
added refreshing query after clearing filters to prevent getting results of previous query